microsoftgraph/microsoft-graph-explorer-v4
# Fix: Feedback rating

## Overview
Fixes #1667 

### Demo
![image](https://user-images.githubusercontent.com/45680252/165693730-c6823c73-89fb-40c9-b2c4-c8f8bfcbc6c4.png)


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
Check out this branch
Use the app for a minute
A feedback popup will appear with the new ratings
